frontend/src/theme/custom.scss

```diff
@@ -145,3 +145,20 @@ ion-item.item-has-focus {
     border-color: var(--ion-color-tertiary);
   }
 }
+/**
+ * Customization of the alert in the dropdownmenu of the finalize-page
+*/
+.custom-class {
+  .cancel-button {
+    color: blue;
+  }
+  .finalize-button {
+    color: red;
+  }
+}
```

Comment:
Please don't hard-code these colors here. If you want to use a red color, then use `var(--ion-color-warning)` (or the secondary color, which is at least in our case red as well).

Also, maybe make everything a bit more generic: Just create the css classes like so:

``` css
.custom-alert-button-colors {
  .color-secondary {
    color: var(--ion-color-secondary);
  }
}
```

And finally: I know that in the mockups, one button is blue. But I think it would be ok to keep the default color.